Makefile

```diff
@@ -159,16 +159,16 @@ NOTES := $(abspath $(TOOLS_BIN_DIR)/$(NOTES_BIN))
 
 # Registry / images
 TAG ?= dev
-ARCH ?= $(shell go env GOARCH)
+ARCH ?= linux/$(shell go env GOARCH)
 ALL_ARCH = amd64 arm64
```

Comment:
this looks redundant now?

Reply:
to reduce time of image building in our e2e, we will pass only current architecture to buildx

Reply:
I am not getting it, usage of `ALL_ARCH` env var is on line 366 only and that is being removed in this PR

Reply:
sorry, I thought the comment is about 162

Reply:
I removed it